var/spack/repos/builtin/packages/protobuf/pkgconfig.patch

```diff
@@ -0,0 +1,64 @@
+diff --git a/cmake/CMakeLists.txt b/cmake/CMakeLists.txt
```

Comment:
this file is the patch in https://github.com/google/protobuf/pull/3403

we can update it as the review processes there